fiftyone/utils/torch.py

```
@@ -1461,6 +1470,237 @@ def _is_string_array(targets):
+            raise ValueError(
+                "This class only works with persistent datasets. Please set your dataset to be persistent."
+            )
+
+        if samples.media_type == fomd.GROUP:
+            raise NotImplementedError(
+                "Not compatible with grouped datasets yet. If you want this feature, please let us know!"
+            )
+
+        start_method = multiprocessing.get_start_method(allow_none=True)
+        if start_method not in ["spawn", "forkserver"]:
+            warnings.warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)